crates/hyperswitch_connectors/src/utils.rs

```diff
@@ -1529,3 +1531,37 @@ pub trait ForeignTryFrom<F>: Sized {
 
     fn foreign_try_from(from: F) -> Result<Self, Self::Error>;
 }
+
+#[derive(Debug)]
+pub struct QrImage {
+    pub data: String,
+}
+pub(crate) const QR_IMAGE_DATA_SOURCE_STRING: &str = "data:image/png;base64";
+pub(crate) const BASE64_ENGINE: base64::engine::GeneralPurpose =
```

Comment:
use `BASE64_ENGINE` which is already present at `crates/common_utils/src/consts.rs` and remove this declaration